numpy/random/mtrand.pyx

```diff
@@ -266,10 +272,6 @@ cdef class RandomState:
             If `legacy` is False, or the BitGenerator is not NT19937, then
```

Comment:
```suggestion
            If `legacy` is False, or the BitGenerator is not MT19937, then
```

Reply:
It is spaced with 12, which I think is right.

Reply:
Fixed N->M, but leading space seems right at 12.

Reply:
Leading space is a GitHub bug, hitting "commit suggestion" doesn't actually add it.